modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3252,6 +3253,13 @@ def iloc_mut(partition, row_internal_indices, col_internal_indices, item):
         return self.__constructor__(new_modin_frame)
 
     def sort_rows_by_column_values(self, columns, ascending=True, **kwargs):
+        # Our algebra sort is only implemented for Engines that support virtual partitioning.
+        if Engine.get() in ["Ray", "Dask"]:
+            kwargs.pop("ascending", False)
```

Comment:
how can `kwargs` contain `ascending` if it is specified as an explicit argument already?